lib/membrane_live/webinars/webinar.ex

```diff
@@ -0,0 +1,43 @@
+defmodule MembraneLive.Webinars.Webinar do
+  @moduledoc """
+  Struct for webinar
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          description: String.t(),
+          moderator_link: String.t(),
+          presenters: list,
+          start_date: NaiveDateTime.t(),
+          title: String.t(),
+          viewer_link: String.t()
+        }
+
+  schema "webinars" do
+    field(:description, :string)
+    field(:moderator_link, :string)
+    field(:presenters, {:array, :string})
+    field(:start_date, :naive_datetime)
+    field(:title, :string)
+    field(:viewer_link, :string)
+
+    timestamps()
+  end
+
+  @spec changeset(
+          {map, map}
+          | %{
+              :__struct__ => atom | %{:__changeset__ => map, optional(any) => any},
+              optional(atom) => any
+            },
+          :invalid | %{optional(:__struct__) => none, optional(atom | binary) => any}
+        ) :: Ecto.Changeset.t()
```

Comment:
As we removed dialyzer, you can go through files and remove these which aren't useful in your opinion.

Reply:
OK, will do 🏃‍♂️

Reply:
The problem with this is that `mix credo` pipeline won't pass because of public functions without `@spec`. Should I comment those functions with `#credo:ignore-next-line`? #️⃣